src/components/users/signup-social/ko/signupSocial.html

```diff
@@ -1 +1,12 @@
+<!-- ko if: identityProvider -->
 <signup-social-runtime></signup-social-runtime>
+<!-- /ko -->
+
+<!-- ko ifnot: identityProvider -->
+<div class="description-text">This widget will display a sign-up form when you configure <a
+    href="https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-aad" target="_blank"
```

Comment:
You should be able to add links to https://aka.ms - it's better to do that because the documentation moves from time to time and if we have a short link, we won't need to update application, only the short link itself in aka.ms tool. At least it worth moving it from the markup to a constant, and reference it here.

I have added two short links:
https://aka.ms/apim-how-to-aad
https://aka.ms/apim-how-to-aadb2c

Reply:
`alt` attribute is useful only in <img> tag.